app/renderer/components/common/browserButton.js

```diff
@@ -188,8 +192,19 @@ const styles = StyleSheet.create({
     //
     // TODO: remove !important and check advancedSettings.js
     // once preferences is fully refactored
-    marginRight: '4px !important',
-    marginLeft: '4px !important'
+
+    // 'mm' assures theoretically the equal width of margin
+    // on every platform with any display monitor resolution.
+    // 2 mm = 1/12 inch
+    marginLeft: '2mm !important',
```

Comment:
I have a strong guess that most monitors aren't yet in the same high-res as printers to get absolute units with the same precision. Absolute units are basically made for printed pages.

If you're interested, please check https://www.w3.org/Style/Examples/007/units.en.html and lmk if that changed your mind or, if not, lmk your thoughts.